infra/build-image/setup-repo.sh

```diff
@@ -26,14 +26,24 @@ set -e && [ -n "$DEBUG" ] && set -x
 DIR=$(dirname $(readlink -f "$0"))
 
 function usage() {
-echo "Usage: $0 -p pull-request OR -b github_user/branch" 1>&2
+echo "Usage: $0 -p pull-request OR -b github_user/branch OR -t tag OR -c commit" 1>&2
 exit 1
 }
 
-while getopts "p:b:h" flag
+while getopts "t:c:p:b:h" flag
```

Comment:
Should we include some validation that only one of the flags are provided?

Reply:
I add validation to allow only one of flags provided.